Packs/FortiSIEM/IncidentFields/incidentfield-FortiSIEM_Event_Type.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
@OfekMQmasters Seems like this field is not needed since there is already an "Event Type" field (shortText) under "CommonTypes" pack, cliname - `eventtype`. Please use the existing field.